packages/admin-ui/src/filters/__stories__/filters-searchable.stories.tsx

```diff
@@ -107,30 +107,34 @@ export function MultiSearch() {
   )
 }
 
-// problematic, there should be a way to update fulllist async
-// this is the case where the initial value is fetched but the
-// search is uncontrolled
 export function Async() {
-  const [fullList, setFullList] = useState<
-    Array<{ id: string; label: string }>
-  >([])
-
-  const filterState = useFilterMultipleState({ fullList })
+  const filterState = useFilterMultipleState()
+  const { setFullList, setStatus, combobox, status } = filterState
 
   useEffect(() => {
-    api('', 5000).then((res: any[]) =>
+    setStatus('loading')
+    api('', 5000).then((res: any[]) => {
       setFullList(res.map((i) => ({ id: i.id, label: i.country })))
-    )
+      setStatus('ready')
+    })
   }, [])
 
+  useEffect(() => {
```

Comment:
this use effect is only necessary because after you set status for the first time it becomes controlled forever (you can't unset it) so you don't get automatic updates. Maybe I should allow status undefined, or have a status 'auto' idk. On the other hand I don't know if it's nice to mix controlled and uncontrolled behavior.

Opinions welcome